atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/prismagent/protocols/outOfBand/OutOfBandInvitation.kt

```diff
@@ -32,8 +32,8 @@ constructor(
     @Serializable
     data class Body(
         @SerialName(GOAL_CODE)
-        val goalCode: String?,
-        val goal: String?,
+        val goalCode: String? = "",
+        val goal: String? = "",
```

Comment:
Should these values be null and with `@EncodeDefault` or as per the conversation on slack it should be empty string?!